ha/core/cluster/cluster_manager.py

```diff
@@ -321,7 +321,7 @@ def _validate_permissions(self) -> None:
                 raise HAInvalidPermission(
                             f"User {user} does not have necessary permissions to execute this CLI")
 
-            # The user name "hauser"; which is part of the "haclient" group;
+            # TODO : Check if user needs to be changed to hauser
             # is used by HA.
```

Comment:
remove all these commented lines and only keep the TODO

Reply:
Removed all comments